chef/cookbooks/network/recipes/default.rb

```diff
@@ -509,4 +511,26 @@ def kill_nic(nic)
       end
     end
   end
+
+  if ovs_setup_once
+    # If we're using an ovs-bridge somewhere, enable wicked-nanny to be started
+    # for the next boot.
+    # Note: There's no need to restart wicked here as all interfaces
+    # should be correctly configure at this point.
+    template "/etc/wicked/local.conf" do
+      source "wicked-local.conf.erb"
+      owner "root"
+      group "root"
+      mode "0644"
+      variables(
+        use_nanny: true
+      )
```

Comment:
Hm, that code is not SLES11 specific. It's for everything that has wicked. (SLE12 and newer, including openSUSE if we supported that)